cmd/utils/flags.go

```diff
@@ -211,6 +212,29 @@ var (
 		Name:  "db.leveldb.no-buffer-pool",
 		Usage: "Disables using buffer pool for LevelDB's block allocation",
 	}
+	DynamoDBTableNameFlag = cli.StringFlag{
+		Name:  "db.dynamo.tablename",
+		Usage: "Specifies DynamoDB table name. If it is not set, a new name will be created. (Set dbtype to use DynamoDB)",
```

Comment:
How about terminating a node when the table name is not specified?
Since using dynamoDB consumes users' AWS credits (or money), we should be careful.

Reply:
How about not allowing to skip the `db.dynamo.tablename`?

Reply:
review reflected :) PTAL